homeassistant/components/yamaha_musiccast/select.py

```diff
@@ -8,6 +9,7 @@
     MusicCastCapabilityEntity,
     MusicCastDataUpdateCoordinator,
 )
+from homeassistant.components.yamaha_musiccast.const import DEVICE_CLASS_MAPPING
```

Comment:
This should be a relative import:
```suggestion
from .const import DEVICE_CLASS_MAPPING
```

Reply:
You may need to rebase also as there is a conflict

Reply:
done!